armeria/src/main/resources/logback.xml

```diff
@@ -8,7 +8,18 @@
     </encoder>
   </appender>
 
-  <logger name="com.linecorp.armeria" level="info"/>
+  <appender name="ACCESS" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <pattern>[%X{traceId}/%X{spanId}] %msg%n</pattern>
```

Comment:
so here are the MDC provided keys. Without explicit decoration, these are blank as there's no trace in context anymore.